sdk/formrecognizer/azure-ai-formrecognizer/src/test/resources/session-records/DocumentAnalysisAsyncClientTest.testHtmlDocumentPrebuiltRead[1].json

```diff
@@ -0,0 +1,67 @@
+{
+  "networkCallRecords" : [ {
+    "Method" : "POST",
+    "Uri" : "https://REDACTED.cognitiveservices.azure.com/formrecognizer/documentModels/prebuilt-read:analyze?stringIndexType=utf16CodeUnit&api-version=2023-02-28-preview&features=&queryFields=",
+    "Headers" : {
+      "User-Agent" : "azsdk-java-azure-ai-formrecognizer/4.1.0-beta.1 (11.0.9; Windows 10; 10.0)",
+      "x-ms-client-request-id" : "8c54e545-929d-46ce-9a60-292fe9c41749",
+      "Content-Type" : "application/octet-stream"
+    },
+    "Response" : {
+      "x-ms-region" : "East US",
+      "x-envoy-upstream-service-time" : "59",
+      "Strict-Transport-Security" : "max-age=31536000; includeSubDomains; preload",
+      "x-content-type-options" : "nosniff",
+      "apim-request-id" : "401a0f65-eddc-4e0f-8015-4e84d0977ab1",
+      "retry-after" : "0",
+      "Content-Length" : "0",
+      "StatusCode" : "202",
+      "Operation-Location" : "https://shafangfr.cognitiveservices.azure.com/formrecognizer/documentModels/prebuilt-read/analyzeResults/401a0f65-eddc-4e0f-8015-4e84d0977ab1?api-version=2023-02-28-preview",
```

Comment:
Is `Operation-Location` still not redacted?

Reply:
This is still the old test setup.